### DISCUSSION TITLE
Actually Additions Item Lasers void Upgrades if stack size == 0

### DISCUSSION THREAD

**Discussion by WaitingIdly** [ACGaming/UniversalTweaks#421] [Bugfix Suggestions]
When applying Actually Additions upgrades, the stack is reduced and then attempts apply the upgrade to the Laser. However, if there is only 1 item in the stack, it reduces the stack to "air" and then applies "air". It functions properly with > 1.

Source code for this issue: https://github.com/Ellpeck/ActuallyAdditions/blob/1.12.2/src/main/java/de/ellpeck/actuallyadditions/mod/blocks/BlockLaserRelay.java#L196

**Comment by ACGaming** [comment]
Implemented in 995892591d2ea0ea2ddb1e3771e2de1fa92d9690
